src/components/scenes/GettingStartedScene.tsx

```diff
@@ -136,9 +136,15 @@ export const GettingStartedScene = (props: Props) => {
     visitPasswordScene()
   })
 
-  const handlePressSignUp = useHandler(() => {
-    dispatch(logEvent('Signup_Welcome'))
-    visitNewAccountScene()
+  const handlePressGetStartedNext = useHandler(() => {
+    // If we're at the last slide, navigate to account creation
+    if (swipeOffset.value >= sections.length) {
+      dispatch(logEvent('Signup_Welcome'))
+      handleFinalSwipe()
```

Comment:
Why is this case necessary to handle here?

Reply:
It needs to behave the same as a swipe - progressing past the last slide should
complete the flow.